README.md

```diff
@@ -50,6 +50,16 @@ For example, this line might begin to look like this:
 plugins=(git bundler osx rake ruby)
 ```
 
+##### With Conditionals
+
+You may want to control when and/or how plugins should be enabled.
+
+For example, if you want the `tmux-autoattach` plugin to only run on SSH sessions, you could employ a trivial conditional that checks for the `$SSH_TTY` variable. Just make sure to remove the plugin from the larger plugin list.
+
+``` bash
+[ "$SSH_TTY" ] && plugins+=(tmux-autoattach)
+```
+
```

Comment:
Maybe we can also add the same explanation in [templates/bashrc.osh-template](https://github.com/ohmybash/oh-my-bash/blob/master/templates/bashrc.osh-template). What do you think?